examples/docs_projects/project_ask_ai_dagster/pyproject.toml

```diff
@@ -13,7 +13,6 @@ dependencies = [
     "langchain-openai",
     "langchain-chroma",
     "dagster",
-    "dagster-dg-cli",
```

Comment:
Btw @dehume , do we not need this dependency?

I noticed recently on a newly scaffolded `quickstart_etl` project that I didn't have `dg` available to me on PATH.

Reply:
I missed that one. Trying to keep those to just the dev dependencies section

Reply:
Gotcha!

Reply:
I have been just keeping `dagster-webserver` and `dagster-dg-cli` as dev dependencies as the create project initialization does. Sine they are more important for development